PythonScript/src/CreateWrapper.py

```diff
@@ -503,15 +494,15 @@ class PythonCompatibleStrBuffer
 					(v["ReturnType"], v["Param1Type"], v["Param1Name"], v["Param2Type"], v["Param2Name"], body) = specialCases[v["Name"]]
 				else:
 					sig = mapSignature((v["Param1Type"], v["Param1Name"], v["Param2Type"], v["Param2Name"]))
-					returnType = "int"
+
 					if sig is not None:
-						print '{:<45} {} ==>> {}'.format(v["Name"], v["ReturnType"], sig[0])
+						# print '{:<45} {} ==>> {}'.format(v["Name"], v["ReturnType"], sig[0])

```

Comment:
Is this just debug output? In this case a description might be helpfull or could it be removed?